backend/apps/ml_triage/services/github_collector.py

```
@@ -0,0 +1,139 @@
+
+    def _process_reactions(self, issue: Issue, reactions_data: Dict):
+        """
+        Process reactions for an issue.
+        """
+        if not reactions_data:
+            return
+        
+        # Simplified reaction processing
+        for reaction_type, count in reactions_data.items():
+            if isinstance(count, int) and count > 0:
+                for i in range(count):
```

Ruff (0.15.21)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)